PlayCover/de.lproj/Localizable.strings

```diff
@@ -1,44 +1,93 @@
-"To do this you need to reboot, enter into Recovery mode, open Terminal and type" = "Um dies zu tun, musst Du deinen Mac neu starten, in den Wiederherstellungsmodus wechseln";
-"Your nvram params are not valid. Execute following command in Terminal and restart Mac" = "Deine; nvram-Parameter sind ungültig. Führe den folgenden Befehl im Terminal aus und starte den Mac neu";
-"Your system is not ready to run games! Watch video for easy setup." = "Dein System ist nicht bereit, Spiele auszuführen! Sieh Dir das Video zur einfachen Einrichtung an.";
-"Your system setup is correct." = "Deine Systemkonfiguration ist korrekt.";
-"About app" = "Über die App";
-"This .IPA can't be decrypted on this Mac. Download .ipa from AppDb.to or our Discord. You can also try to use 'Alterntaive decrypt' checkbox" = "Diese .IPA kann auf diesem Mac nicht entschlüsselt werden. Lade Dir eine .ipa von AppDb.to oder unserem Discord herunter. Du kannst auch versuchen, 'Alterntaive Entschlüsselung' zu verwenden' Kontrollkästchen";
-"This .IPA is corrupted. It doesn't contain Info.plist." = "Diese .IPA ist beschädigt. Sie enthält keine Info.plist.";
-"It is impossible to decrypt .IPA with SIP disabled. Please, enable it." = "Es ist unmöglich, .IPA mit deaktiviertem SIP zu entschlüsseln. Bitte aktiviere es.";
-"Please reinstall PlayCoverApp" = "Bitte installiere die PlayCoverApp neu";
-"Make sure you don't disallowed PlayCover to access files in Settings - Secuirity & Privacy" = "Stelle sicher, dass Du PlayCover den Zugriff auf Dateien in den Einstellungen – Sicherheit und Datenschutz nicht untersagt hast";
-"This .IPA is courrupted.Try to use another .ipa" = "Diese .IPA ist beschädigt.Versuch, eine andere .ipa . zu verwenden";
-"You have not performed all instructions! Please, perform all instructions in 'Settings' or watch video in pinned messages." = "Du hast nicht alle Anweisungen ausgeführt! Bitte führe alle Anweisungen unter 'Einstellungen' aus oder schau Dir das Video in den angepinnten Nachrichten an..";
-"Get" = "Herunterladen";
-"soundAlert.messageText" = "";
-"soundAlert.informativeText" = "";
-"soundAlert.successText" = "";
+"app.add" = "App hinzufügen";//
+"search.search" = "Search...";
+"app.settings" = "Einstellungen";//
+"app.showInFinder" = "Im Finder anzeigen";//
+"app.openCache" = "Open app data";
+"app.clearCache" = "App-Cache leeren";//
+"app.importkm" = "Import keymapping";
+"app.exportkm" = "Export keymapping";
+"app.delete" = "App löschen";//
+"alert.appCacheCleared" = "App data has been cleared!";
+"alert.kmImported" = "Keymapping imported!";
+"alert.errorImportKm" = "Error occured when importing!";
+"alert.app.delete" = "All app data will be erased. You may need to download app files again. Do you wish to continue?";
+"alert.wrongFileType" = "Falscher Dateityp";//
+"button.OK" = "OK";
+"button.Proceed" = "Proceed";
```

Comment:
Proceed sounds too formal. 'Continue' would be better

Reply:
The button carries out actions that are irreversible and may cause unwanted data loss, and hence I would still prefer "Proceed" over "Continue" as it sounds more serious and that it conveys the same idea to the user